android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="com.coepi">
```

Comment:
Probably not a big deal right now, but we don't have the copei.com domain, just .org and .net, so we should probably switch over to org.coepi before we actually submit anything to Apple or Google.